packages/react-native-renderer/src/ReactNativeEventTelemetryPlugin.js

```diff
@@ -0,0 +1,19 @@
+import {RawEventTelemetryEventEmitter} from 'react-native/Libraries/ReactPrivate/ReactNativePrivateInterface';
+
+const ReactNativeEventTelemetryPlugin = {
+  extractEvents: function(
```

Comment:
Nit: Concise notation.
```suggestion
  extractEvents(
```